Samples~/StaticScriptSample/StaticDependencyEditor.cs

```diff
@@ -0,0 +1,75 @@
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.VspAttribution;
+
+public class StaticScriptSample : EditorWindow
+{
+    static readonly Vector2 s_WindowSize = new Vector2(320,330);
+    
+    public string actionName;
+    public string partnerName;
+    public string customerUid;
+
+    [MenuItem("VSP-Samples/Static Script Information")]
+    public static void Initialize()
+    {
+        var window = GetWindow<StaticScriptSample>();
+        
+        window.titleContent = new GUIContent("Static Script Information!");
+        window.minSize = s_WindowSize;
+        window.maxSize = s_WindowSize;
+    }
+
+    public void OnGUI()
+    {
+        // Implementation information
+        DrawLine(Color.gray);
+        
+        DrawHelpBox("<b><size=11>To implement VSP Attribution service:</size></b>\n" +
+                    "1. Import <color=#ff00ffff>VSPAttribution</color> script into your package\n\n" +
+                    "2. Change the namespace to something else, like:\n" +
+                    "namespace <color=#ff00ffff>UnityEngine.VspAttribution.PartnerName</color> \n\n" +
+                    "3. Add <color=#ff00ffff>SendAttributionEvent</color> call to your login\n\n" +
+                    "or initialization method.", 7);
```

Comment:
I would personally change the colour to something within the Unity design palette. Now it makes me feel it's a hyperlink.